File: content/how-to/assets/examples/listAssets.py
import alpaca_trade_api as tradeapi

api = tradeapi.REST()

# Get a list of all active assets.
active_assets = api.list_assets(status='active')

# Filter the assets down to just those on NASDAQ.
nasdaq_assets = [a for a in active_assets if a.exchange == 'NASDAQ']
print(nasdaq_assets)

File: content/how-to/assets/examples/listAssets.js
const Alpaca = require('@alpacahq/alpaca-trade-api')
const alpaca = new Alpaca({
    paper: true
})

// Get a list of all active assets.
const activeAssets = alpaca.getAssets({
    status: 'active'
}).then((activeAssets) => {
    // Filter the assets down to just those on NASDAQ.
    const nasdaqAssets = activeAssets.filter(asset => asset.exchange == 'NASDAQ')
    console.log(nasdaqAssets)
})
